app/services/service_provider_seeder.rb

```diff
@@ -12,18 +12,18 @@ def run
 
     service_providers.each do |issuer, config|
       next unless write_service_provider?(config)
-      ServiceProvider.find_or_create_by!(issuer: issuer) do |sp|
-        cert_pems = Array(config['certs']).map do |cert|
-          cert_path = Rails.root.join('certs', 'sp', "#{cert}.crt")
-          cert_path.read if cert_path.exist?
-        end.compact
 
+      cert_pems = Array(config['certs']).map do |cert|
+        cert_path = Rails.root.join('certs', 'sp', "#{cert}.crt")
+        cert_path.read if cert_path.exist?
+      end.compact
+
+      ServiceProvider.find_or_create_by!(issuer: issuer) do |sp|
```

Comment:
**question:** what do you think about adding a comment here for future us so we know what the difference between the two blocks is?

Reply:
Gonna try without the comment for now but maybe I'll come to regret this decision next time we make this mistake lol

Reply:
Living on the edge 😂